core/jobs.py

```diff
@@ -685,33 +892,64 @@ class BaseRealtimeDatastoreClassForContinuousComputations(
     the 0 or 1 indicates the realtime layer that the entity is in.
 
     NOTE TO DEVELOPERS: Ensure that you wrap the id with get_realtime_id()
-    when doing creations, gets, puts and queries, in order to ensure that the
+    when doing creations, gets, puts, and queries; in order to ensure that the
     relevant layer prefix gets appended.
     """
     realtime_layer = ndb.IntegerProperty(required=True, choices=[0, 1])
 
     @classmethod
     def get_realtime_id(cls, layer_index, raw_entity_id):
-        """Returns an ID used to identify the element with the given entity id
-        in the currently active realtime datastore layer.
+        """Returns a valid id for identifying the entity with the given raw id
+        from the given active realtime datastore layer.
+
+        Args:
+            layer_index: int. The realtime layer id, should be either 0 or 1.
+            raw_entity_id: int. The id of some entity wishing storage in the
```

Comment:
Nit: The id of an entity that is to be stored in the given realtime layer.

("wishing" implies that the entity has agency, rather than the process which handles it.)